mysql-test/suite/galera/t/mysql-wsrep-372.test

```diff
@@ -0,0 +1,311 @@
+#
+# Test the case where a Galera replica to async replication uses replicate-do-db/replicate-do-table.
+# Cluster originating replication should not be affected by this.
+#
+
+--source include/have_log_bin.inc
+--source include/big_test.inc
+
+--connect node_2a, 127.0.0.1, root, , test, $NODE_MYPORT_2
```

Comment:
Do we use this connection somewhere?

source count_sessions.inc/wait_until_count_sessions.inc

Reply:
Yes. node1 isn't part of the cluster, so we can't initialize galera on it. We have to connect to node2 first, and initialize the cluster there.